sklearn/linear_model/logistic.py

```diff
@@ -1128,6 +1128,18 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
             In SciPy <= 1.0.0 the number of lbfgs iterations may exceed
             ``max_iter``. ``n_iter_`` will now report at most ``max_iter``.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> X, y = load_iris(return_X_y=True)
+    >>> clf = LogisticRegression(random_state=0).fit(X, y)
+    >>> clf.score(X, y) # doctest: +ELLIPSIS
+    0.96
+    >>> clf.predict_proba(X[:2,:]) # doctest: +ELLIPSIS
```

Comment:
I think there's flake8 error here (should be ``clf.predict_proba(X[:2, :])``)? I think we should avoid flake8 errors in these examples, though they'll not be detected by CI.

Reply:
Done.